setup.py

```diff
@@ -18,15 +18,15 @@
     url='http://councilmatic.org/',
     author='DataMade, LLC',
     author_email='info@datamade.us',
-    install_requires=['requests==2.7.0',
-                      'pytz==2015.4',
-                      'django-haystack==2.5.0',
+    install_requires=['requests>=2.7.0,<2.8',
+                      'pytz>=2015.4',
```

Comment:
Is there any hazard in setting this without an upper boundary? Could a newer version of pytz (and psycopg below) cause issues?

Reply:
This is a case of making trade-offs between different [type of errors](https://en.wikipedia.org/wiki/Type_I_and_type_II_errors).

If we don't specify an upper boundary, it's possible that a future version will come out which will not be compatible with this library. This is a type II error.

If we do specify an upper boundary, it's possible that there are many versions that are higher than the specified boundary that are also compatible.

So, yes there is a hazard, but there are also benefits.

Personally, for libraries, I think it's better to make type II errors. If we don't know that a higher version is incompatible, we should not assume that it is incompatible. 

That's a larger departure from our current practice, which is why I did not do that in this PR.

In this case, I did not set an upper boundary for pytz, because they do not use semantic versioning so using there's no real principled upper boundary to set.

For pscyopg2, I know this is a library that strives for maintaining backwards compatibility so I feel pretty comfortable not setting an upper bound.

Reply:
All right! Thanks for the reference point, too.